waspc/src/Wasp/Generator/WebAppGenerator/OperationsGenerator.hs

```diff
@@ -20,10 +22,14 @@ import qualified Wasp.AppSpec.Query as AS.Query
 import Wasp.Generator.FileDraft (FileDraft)
 import Wasp.Generator.Monad (Generator)
 import qualified Wasp.Generator.ServerGenerator as ServerGenerator
+import Wasp.Generator.ServerGenerator.Common (serverSrcDirInServerRootDir)
+import Wasp.Generator.ServerGenerator.OperationsG (operationFileInSrcDir)
```

Comment:
Maybe import these qualified in a way that makes it clear this is coming from Server. For `serverSrcDirInServerRootDir` it is clear, but for `operationFileInSrcDir` it is not.